bc_obps/registration/utils.py

```diff
@@ -110,13 +110,6 @@ def raise_401_if_user_not_authorized(
                 raise HttpError(401, UNAUTHORIZED_MESSAGE)
 
 
-def get_current_user_approved_user_operator_or_raise(user: User) -> Optional[UserOperator]:
-    user_operator = user.get_approved_user_operator()
-    if not user_operator:
-        raise HttpError(401, UNAUTHORIZED_MESSAGE)
-    return user_operator
-
-
```

Comment:
I was debating whether to move utils into services, thoughts?

Reply:
Thanks. I left the file ones in here because they seemed more helper-y than Document model-y, but if you have strong feelings happy to move them (I was at a bit of a loss here)

Reply:
💯